.circleci/config.yml

```diff
@@ -12,6 +12,9 @@ jobs:
           cd /home/linuxbrew/.linuxbrew/Homebrew
           git fetch origin --unshallow --tags
           git reset --hard origin/master
+          cd /home/linuxbrew/.linuxbrew/Homebrew/Library/Taps/homebrew/homebrew-core
```

Comment:
you could use `cd $(brew --repo linuxbrew/core)` which is a bit shorter

Reply:
True. When something is never expected to change (hopefully), I like the clarity of specifying the precise value. I don't mind either, really.